test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,58 @@
+./test_scripts/API/RAI/012_ATF_N_method_IsWrongType_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_wrong_type
+./test_scripts/API/RAI/013_ATF_N_method_IsInvalidCharacter_Space_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_invalid_character_space
+./test_scripts/API/RAI/014_ATF_N_method_IsInvalidCharacter_Tab_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_invalid_character_tab
+./test_scripts/API/RAI/015_ATF_N_method_IsInvalidCharacter_NewLine_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_method_is_invalid_character_new_line
+./test_scripts/API/RAI/016_ATF_N_resultCode_IsMissed_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_resultcode_missing
+./test_scripts/API/RAI/017_ATF_N_resultCode_IsNotExist_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_resultcode_is_not_existing
+./test_scripts/API/RAI/018_ATF_N_resultCode_IsWrongType_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_resultcode_is_wrong_type
+./test_scripts/API/RAI/019_ATF_N_resultCode_INVALID_DATA_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_resultcode_invalid_data
+./test_scripts/API/RAI/020_ATF_N_resultCode_DATA_NOT_AVAILABLE_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_resultcode_data_not_available
+./test_scripts/API/RAI/021_ATF_N_resultCode_GENERIC_ERROR_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_resultcode_generic_error
+./test_scripts/API/RAI/022_ATF_N_message_IsMissed_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_message_is_missing
+./test_scripts/API/RAI/023_ATF_N_message_IsLowerBound_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_message_is_lower_bound
+./test_scripts/API/RAI/024_ATF_N_message_IsUpperBound_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_message_is_upper_bound
+./test_scripts/API/RAI/025_ATF_N_message_IsOutUpperBound_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_json_message_message_is_out_upper_bound

```

Comment:
@OVikhrov, the idea of script is to send errorcode "INVALID_DATA"(11) and to check that with valid / invalid message, the UI.GetCapabilities will be invalidated by SDL.